bnelson777/id.ly
# IL-157: Android suppress promise rejection warning

Note: Deep link server may not be up at the moment. This ticket may break that.
 
# Testing
- Boot into the home screen using react-native run-android and ensure no warnings pop up.
- Create an identity on device 1
- Create an identity on device 2
- Share each identity between both devices
- Send a message from device 1 to device 2
- Open link in email, decrypt ,and ensure the message shows up in the message thread.
